src/App/Pages/Vault/CipherAddEditPage.xaml

```diff
@@ -292,7 +292,7 @@
                             StyleClass="box-row-button, box-row-button-platform"
                             Text="{Binding Source={x:Static core:BitwardenIcons.Clone}}"
                             Command="{Binding CopyCommand}"
-                            IsVisible="{Binding HasTotpValue}"
+                            IsVisible="{Binding Cipher.ViewPassword}"
```

Comment:
🤔 Shouldn't this be `HasTotpValue && Cipher.ViewPassword` in the VM and binded here?

Reply:
You're right, not sure how I missed that.  Fixed.